GameData/ROEngines/PartConfigs/Viking2_FRE.cfg

```diff
@@ -0,0 +1,81 @@
+PART
+{
+	module = Part
+	name = ROE-Viking2
+	author = Zarbizaure, capkirk
+
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+
+	MODEL
+	{
+		model = ROEngines/Assets/FRE/VikingEarly/VikingEarly
+	}
+
+	rescaleFactor = 1
+	attachRules = 1,1,1,0,0
+	node_stack_top = 0.0, 1.61, 0.0, 0.0, 1.0, 0.0
+	node_stack_bottom = 0.0, -1.03, 0.0, 0.0, -1.0, 0.0
+	node_attach = 0.0, 1.61, 0.0, 0.0, 1.0, 0.0
+
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.8 // engine nozzles are good at radiating.
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 2
+
+	title = Viking (2/2B)
+	manufacturer = Snecma
+	description = Early Viking engines used on Ariane 1, 2 and 3. Thanks to the magic of hypergolic propellant, Vikings engines were able to achieve a substainable amount of thrust at all altitude. Includes configs for Viking 2 and 2B. [1.5m]
+
+	engineType = Viking
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
```

Comment:
Can you make these parts [and your other PRs] use the tool in https://github.com/KSP-RO/ROEngines/blob/PatchingFixes/GameData/ROEngines/ROEEngineDefaults.cfg?